### ISSUE TITLE
Update package.json main attribute

### ISSUE THREAD

**Issue by Knorcedger** [iamstarkov/generator-travis#12]
Hello,

i'm trying to create a composite generator that is using this one, but its not able to even find the 'generator-travis' module. After a couple hours banging my head, I found out that if I change the attribute **main** in your package.json to the correct path (generators/app/index.js) everything is fine.

Tnx!

**Comment by SBoudrias** [comment]
@Knorcedger you want to use [composeWith](https://yeoman.io/authoring/composability.html).

Then to get the path you want `require.resolve('generator-travis/app')`.

No need for a main field as it currently doesn't mean anything in the Yeoman context.

**Comment by Knorcedger** [comment]
@SBoudrias No, it says again Error: Cannot find module 'generator-travis/app'

That's the code I'm using. Not sure why changing main attribute fixes it, but it does :P

```
travis: function() {
  this.composeWith('knr-generator:travis', {}, {
    local: require.resolve('generator-travis/app')
  });
}
```

**Comment by SBoudrias** [comment]
Did you `npm i --save generator-travis`?

**Comment by SBoudrias** [comment]
Also, you actually want `require.resolve('generator-travis/generators/app')`

**Comment by Knorcedger** [comment]
Of course. I deleted the node_modules and installed again and then did the change you told me.

My code is here -> https://github.com/Knorcedger/generator-knr-generator

EDIT: Yes, that works, but why does changing main also work?

**Comment by iamstarkov** [comment]
@SBoudrias thank you for help.

@Knorcedger i have [composability](https://github.com/iamstarkov/generator-travis#composability) section in readme, where you find instructions how to use this package:
![screen shot 2016-01-14 at 18 23 31](https://cloud.githubusercontent.com/assets/559321/12331904/f18ba64e-baeb-11e5-9a6a-9fd78d7ed66b.png)

**Comment by Knorcedger** [comment]
My bad, I didn't notice them, sorry :(

**Comment by iamstarkov** [comment]
no worries, its fine. glad that you decided to use this project

**Comment by iamstarkov** [comment]
i will add this stuff

**Comment by iamstarkov** [comment]
fixed in v1.2.3
